.github/workflows/cd.yml

```
@@ -0,0 +1,76 @@
+on:
+  push:
+    branches: [ "develop", "feat/#5/cicd" ]
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: JDK 21 설정
+        uses: actions/setup-java@v3
```

actionlint (1.7.10)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)